.github/workflows/main.yml

```
@@ -466,3 +466,84 @@ jobs:
           path: |-
             ${{ github.workspace }}/aws-cdk/dist/
             ${{ github.workspace }}/aws-cdk/**/dist/
+  push-go-runtime:
```

Nitpicking, but can we add a blank line above this one?

```suggestion

  push-go-runtime:
```